src/main/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentController.java

```diff
@@ -97,8 +97,8 @@ public CaseAssignmentController(CaseAssignmentService caseAssignmentService, Mod
             message = AuthError.UNAUTHORISED_S2S_SERVICE
         ),
         @ApiResponse(
-            code = 500,
-            message = CaseAssignmentService.CASE_COULD_NOT_BE_FETCHED
+            code = 404,
+            message = ValidationError.CASE_NOT_FOUND
```

Comment:
Should this PR be flagged as a breaking change if CaseAssignment is already live and this response is changing?

Reply:
well.. it's 500 to 404 => i don't think any client is handling 500 response in this scenario. (not documented in the published Swagger). To me, UI / service team never send any random caseNumbers.

Reply:
Agreed thanks